HW03/aleksei_zhuchkov/aleksei_zhuchkov_hw_03_t_01.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(data_log)
+    array_data_log = data_log.split("\n")
+    array_data_error = array_data_log.select { |el| el.to_s.downcase.include?('error') }
+    array_data_error[0] || ''
```

Comment:
```suggestion
    data_log.split("\n").find { |line| line.downcase.include?('error') }. to_s
```
or (/\n/) instead of ("\n")

Read about the differences between select and find and choose the right one.

Reply:
Tnkx for suggestion, i have done it https://github.com/labs-ruby/homeworks-2022/pull/63/commits/fb17449697f0debd8eec44892f1ef48650fb9fa6

Reply:
Tnkx for suggestion, i have done it https://github.com/labs-ruby/homeworks-2022/pull/63/commits/fb17449697f0debd8eec44892f1ef48650fb9fa6